### PR TITLE
add sd-cmd feature.

### PR DESCRIPTION
**Pull request by yokawara**
## Context
Currently we can't share binaries or scripts across multiple containers to use some commands in all containers. We need a single interface for executing a versioned command during a build.

## Objective
This PR adds feature file for sd-cmd functional testing.

## References
https://github.com/screwdriver-cd/screwdriver/blob/master/design/commands.md

### CODE REVIEW

File: features/sd-cmd.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    (via remote binary, docker image, or habitat package) during a build.
+
+    Background:
+        Given an existing pipeline with <image> image and a command:
+            | image         | namespace   | command        | version |
+            | golang:latest | sd-cmd-test | sample-command | 1.0.0   |
+
+    Scenario Outline: Execute a command of habitat format
+        Given command specifications as mode: <mode>, package: <package>, binary: <binary>
+        When the command package mode is <mode>
```
Comment (tkyi) [inline features/sd-cmd.feature]: probably can get rid of one of the `mode`s since it's on line 16 and 17

File: features/sd-cmd.feature [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    Scenario: Execute a command of binary format
+        Given command specification as file: "./sample.sh"
+        When execute the command with arguments: "node -v"
+        Then the command finishes successfully
+
+    Scenario: Publish a command
+        Given command specification file
+        When execute "publish"
+        Then the command to be successfully published
```
Comment (tkyi) [inline features/sd-cmd.feature]: Then the command is published successfully

File: features/sd-cmd.feature (hunk 1 above)
Comment (yokawara) [reply features/sd-cmd.feature]: Thanks, I removed this line.

File: features/sd-cmd.feature (hunk 2 above)
Comment (yokawara) [reply features/sd-cmd.feature]: fixed

**Comment by yokawara** [comment]
I rebased this branch to get updates of master.

**Comment by yokawara** [comment]
rebased again.

**Pull request merged**